projects/types.nix

```
@@ -105,17 +105,77 @@ let
             };
             module = mkOption {
               type = nullOr deferredModule;
+              description = ''
+                Contains the path to the NixOS module for the program.
+
+                For modules that reside in NixOS, use:
+
+                ```nix
+                {
+                  module = lib.moduleLocFromOptionString "programs.PROGRAM_NAME";
+                }
+                ```
+
+                If you want to extend such modules, you can import them in a new module:
+
+                ```nix
+                {
+                  module = ./module.nix;
+                }
+                ```
+
+                Where `module.nix` contains:
+
+                ```nix
+                {
```

```suggestion
                { lib, ... }:
                {
```

Otherwise it can't work?